README.md

```diff
@@ -99,68 +117,72 @@ Biometry Exception Types:
 
 If it's a cancellation error, `OperationCanceledException` is thrown.
 
-### GetGapabilites
+#### GetGapabilites
 
 Gets the device's current biometric capabilities.
 
 It will return a struct `BiometryCapabilities` with the detailled device configuration.
 
-### ScanBiometry
+#### ScanBiometry
 
 Attemps to scan the user's biometry.
 
 ``` cs
 await biometryService.ScanBiometry(cancellationToken);
 ```
 
-### Encrypt
+#### Encrypt
 
 Encrypts the value and stores it into the platform secure storage with the given key name.
 
 ``` cs
 await biometryService.Encrypt(cancellationToken, "KeyName", "KeyValue");
 ```
 
-#### Android
+#### - Android
 
 A new `CryptoObject` from `AndroidX.Biometric` is created with a key as a parameter. Then the data will be encrypted and presented to the `BiometricPrompt` manager.
 The final step will encode the data in base64 and store it in App with the shared preferences.
 
-#### iOS
+#### - iOS
 
 The `SecKeyChain` will be used to store a string linked to a key. The OS is in charge of securing the data with biometric authentication during the process.
 
-### Decrypt
+#### Decrypt
 
 Decrypts and gets the data associated to the given key name.
 
 ``` cs
 await biometryService.Decrypt(cancellationToken, "KeyName");
 ```
 
-#### Android
+#### - Android
```

Comment:
Don't prefix headers with `-`.

Here you might not need headers for each platform. Each paragraph is pretty short so you could just start it with something like
> On Android, (...)
>
> On iOS, (...)